specification/TileFormats/PointCloud/README.md

```diff
@@ -171,6 +171,32 @@ This is useful for per-object picking and storing application-specific metadata
 The `BATCH_ID` semantic may have a `componentType` of `UNSIGNED_BYTE`, `UNSIGNED_SHORT`, or `UNSIGNED_INT`. When `componentType` is not present, `UNSIGNED_SHORT` is used.
 The global semantic `BATCH_LENGTH` defines the number of unique `batchId` values, similar to the `batchLength` field in the [Batched 3D Model](../Batched3DModel/README.md) header.
 
+### Draco
+
+Per-point semantics such as position, color, normal, and batch id may may be compressed with [Draco](https://google.github.io/draco/). Draco binary data is stored in the Feature Table binary and a list of the compressed per-point semantics is stored in the Feature Table JSON.
```

Comment:
Maybe add a quick sentence on why this would be used, eg. the glTF extension says

> This allows glTF to support streaming compressed geometry data instead of the raw data